README.md

```diff
@@ -110,12 +93,10 @@ Image Hub supports Envoy-based data planes. It is compatible with and can be dep
 <p style="clear:both;">
 <h2><a href="https://github.com/layer5io/learn-layer5">Learn Layer5</a></h2>
 <a href="https://github.com/layer5io/learn-layer5">
-<picture>
-<source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/99991521/182012730-19bf6776-4ebd-460f-b6bb-75105f2a10bc.svg#gh-dark-mode-only">
-  <img src=".github/assets/images/layer5/5-light-bg.png#gh-light-mode-only"
-style="margin:10px;" width="125px"
+  <img src=".github/assets/images/layer5/5-light-bg.png"
+style="float:left;margin:10px;" width="125px"
```

Comment:
Increase its height so that It aligns with the first line of desc like others .
![Screenshot from 2022-08-27 18-29-03](https://user-images.githubusercontent.com/95926324/187031547-57ef283f-9fd1-47b1-afe4-f91f010d9347.png)

Reply:
@legendarykamal  I tried to add inline styling at Image tag but it doesn't work in readme. So, I replaced bigger logo with smaller layer5 logo and I guess it looks fine. But if not, can you please suggest me how can I add style in readme? (Styling works in VS code markdown editor but not showing output in readme file.)

Reply:
looks fine to me now with bigger logo :+1:  
Github editor is certainly works different in some cases .